app/Models/Contest.php

```diff
@@ -157,6 +157,11 @@ public function isJudgingActive(): bool
         return $this->isJudged() && $this->isVotingOpen() && !$this->show_votes;
     }
 
+    public function getAnonJudgesAttribute()
+    {
+        return $this->getExtraOptions()['anon_judges'] ?? false;
```

Comment:
I think this should be called `show_judges` to be inline with the other `show_x` options